staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -6186,6 +6186,15 @@ type NodeSystemInfo struct {
 	OperatingSystem string `json:"operatingSystem" protobuf:"bytes,9,opt,name=operatingSystem"`
 	// The Architecture reported by the node
 	Architecture string `json:"architecture" protobuf:"bytes,10,opt,name=architecture"`
+	// Swap Info reported by the node.
+	SwapInfo *SwapInfo `json:"swapInfo,omitempty" protobuf:"bytes,11,opt,name=swapInfo"`
```

Comment:
"Info" is a useless suffix.  Why not just call it `Swap`?  That way it's `node.status.nodeInfo.swap.capacity`.  I can't do anything about "nodeInfo" but I can stop us from making it worse.

Reply:
Yup, that's a great point. I've followed `nodeInfo`, but I think your approach is cleaner and more elegant.

Thank you @thockin! PTAL